doc/kubernetes/collector/build.sh

```diff
@@ -24,6 +24,30 @@ helm template --debug ${STARTDIR}/../../../provision/minikube/keycloak \
   --set dbPoolInitialSize=30 \
   --set dbPoolMaxSize=30 \
   --set dbPoolMinSize=30 \
-  --set jvmDebug=false \
   | yq \
   > ${BUILDDIR}/helm/keycloak.yaml
+
+# Those value match the Keycloak on ROSA Benchmark Key Results example
+helm template --debug ${STARTDIR}/../../../provision/minikube/keycloak \
+  --set hostname=minikube.nip.io \
+  --set jvmdebug=false \
+  --set cryostat=false \
+  --set instances=3 \
+  --set cpuRequests=2 \
+  --set cpuLimits=6 \
+  --set memoryRequestsMB=1250 \
+  --set memoryLimitsMB=2250 \
+  --set heapInitMB=512 \
+  --set heapMaxMB=1524 \
+  --set dbPoolInitialSize=30 \
+  --set dbPoolMaxSize=30 \
+  --set dbPoolMinSize=30 \
+  --set infinispan.customConfig=true \
+  --set infinispan.configFile=${STARTDIR}/../../../provision/minikube/keycloak/config/kcb-infinispan-cache-remote-store-config.xml \
```

Comment:
The path is relative:
```suggestion
  --set infinispan.configFile=config/kcb-infinispan-cache-remote-store-config.xml \
```

Reply:
thanks for the suggestion.